u8timeseries/models/theta.py

```diff
@@ -0,0 +1,95 @@
+import statsmodels.tsa.holtwinters as hw
+from ..timeseries import TimeSeries
+from .autoregressive_model import AutoRegressiveModel
+import numpy as np
+import math
+from .statistics import check_seasonality, extract_trend_and_seasonality, remove_seasonality
+
+
+class Theta(AutoRegressiveModel):
+    """
+    An implementation of the Theta method with variable value of the `theta` parameter.
+
+    :param theta: User-defined value for the theta parameter. Default to 0.
+
+    TODO: Maybe? Explain a bit what it is?
+    TODO: Implement OTM: Optimized Theta Method (https://arxiv.org/pdf/1503.03529.pdf)
+    TODO: From the OTM, set theta_2 = 2-theta_1 to recover our generalization - but we have an explicit formula.
+    TODO: Try with something different than SES? They do that in the paper.
+    """
+    def __init__(self, theta: int = 0):
+        super().__init__()
+
+        self.model = None
+        self.coef = 1
+        self.alpha = 1
+        self.length = 0
+        self.theta = theta
+        self.is_seasonal = False
+        self.seasonality = None
+        self.season_period = 0
+
+        # The restriction of values is due to the following facts:
+        # - if theta = 1, then the theta method restricts to a simple exponential smoothing (SES)
+        # - if theta = 2, the formula for self.coef below fails.
+
+        assert self.theta not in [1, 2], 'Theta should be neither 1 nor 2.'
```

Comment:
+1 for raisining instead of asserting (throughout code base).
However still I think theta = 1 could be allowed without raising anything.

Reply:
Yes, I had it as forbidden because I (wrongly) remembered it as something that came from the proof of the formula. But I have looked through it again and there is no problem.

And indeed it is better to use a ValueError.